### ISSUE TITLE
[BUG] Random Textures Entity

### ISSUE THREAD

**Issue by wallxq** [sp614x/optifine#4443]
## Description of Issue
With 1.14.4 and latest optifine my random textures for mobs work but on 1.15.2 and optifine pre30 mobs have standard textures.

## Steps to Reproduce
optifine\random\entity\wolf

in this folder i have wolf.properties : 

skins.2= 2
name.2= ipattern:\u00a7cLv.1\u00a7f Dziki Pies

skins.3= 3
name.3= ipattern:\u00a7cLv.3\u00a7f Wilk

skins.4= 4
name.4= ipattern:\u00a7cLv.13\u00a7f Szary Wilk

and  wolf2.png, wolf3.png, wolf4.png

## OptiFine Version
1.15.2 pre30

## Installation Method
installer

## Additional Information
I use textur pack on server, mobs spawn with plugin mythicmobs.

**Comment by sp614x** [comment]
Is it working on latest 1.16 preview?

**Comment by wallxq** [comment]
Nope, the same problem. I use viaversion for use 1.16 on server 1.15.2. Optifine dont texturing entities with correct nametags if they have color.
